From b4d30e2efecbbd7c944d43326728673b7fe24bad Mon Sep 17 00:00:00 2001
From: stakiar <stakiar@cloudera.com>
Date: Tue, 16 May 2017 15:25:11 -0700
Subject: [PATCH 1115/1164] CLOUDERA-BUILD: CDH-49030: Renaming
 TestQueryLifeTimeHook to ConsoleQueryLifeTimeHook
 so ptest doesn't think TestQueryLifeTimeHook is a
 test

Change-Id: I554d3107361bc1414475a605e6660b4eef1ca348
---
 .../hive/ql/hooks/ConsoleQueryLifeTimeHook.java    |   43 ++++++++++++++++++++
 .../hive/ql/hooks/TestQueryLifeTimeHook.java       |   43 --------------------
 .../queries/clientpositive/query_lifetime_hook.q   |    2 +-
 3 files changed, 44 insertions(+), 44 deletions(-)
 create mode 100644 ql/src/test/org/apache/hadoop/hive/ql/hooks/ConsoleQueryLifeTimeHook.java
 delete mode 100644 ql/src/test/org/apache/hadoop/hive/ql/hooks/TestQueryLifeTimeHook.java

diff --git a/ql/src/test/org/apache/hadoop/hive/ql/hooks/ConsoleQueryLifeTimeHook.java b/ql/src/test/org/apache/hadoop/hive/ql/hooks/ConsoleQueryLifeTimeHook.java
new file mode 100644
index 0000000..e5fbdd5
--- /dev/null
+++ b/ql/src/test/org/apache/hadoop/hive/ql/hooks/ConsoleQueryLifeTimeHook.java
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.ql.hooks;
+
+import org.apache.hadoop.hive.ql.session.SessionState;
+
+public class ConsoleQueryLifeTimeHook implements QueryLifeTimeHook {
+  @Override
+  public void beforeCompile(QueryLifeTimeHookContext ctx) {
+    SessionState.getConsole().getOutStream().println("beforeCompile: " + ctx.getCommand());
+  }
+
+  @Override
+  public void afterCompile(QueryLifeTimeHookContext ctx, boolean hasError) {
+    SessionState.getConsole().getOutStream().println("afterCompile: " + ctx.getCommand());
+  }
+
+  @Override
+  public void beforeExecution(QueryLifeTimeHookContext ctx) {
+    SessionState.getConsole().getOutStream().println("beforeExecution: " + ctx.getCommand());
+  }
+
+  @Override
+  public void afterExecution(QueryLifeTimeHookContext ctx, boolean hasError) {
+    SessionState.getConsole().getOutStream().println("afterExecution: " + ctx.getCommand());
+  }
+}
diff --git a/ql/src/test/org/apache/hadoop/hive/ql/hooks/TestQueryLifeTimeHook.java b/ql/src/test/org/apache/hadoop/hive/ql/hooks/TestQueryLifeTimeHook.java
deleted file mode 100644
index 1f6239c..0000000
--- a/ql/src/test/org/apache/hadoop/hive/ql/hooks/TestQueryLifeTimeHook.java
+++ /dev/null
@@ -1,43 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
- * <p>
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.hadoop.hive.ql.hooks;
-
-import org.apache.hadoop.hive.ql.session.SessionState;
-
-public class TestQueryLifeTimeHook implements QueryLifeTimeHook {
-  @Override
-  public void beforeCompile(QueryLifeTimeHookContext ctx) {
-    SessionState.getConsole().getOutStream().println("beforeCompile: " + ctx.getCommand());
-  }
-
-  @Override
-  public void afterCompile(QueryLifeTimeHookContext ctx, boolean hasError) {
-    SessionState.getConsole().getOutStream().println("afterCompile: " + ctx.getCommand());
-  }
-
-  @Override
-  public void beforeExecution(QueryLifeTimeHookContext ctx) {
-    SessionState.getConsole().getOutStream().println("beforeExecution: " + ctx.getCommand());
-  }
-
-  @Override
-  public void afterExecution(QueryLifeTimeHookContext ctx, boolean hasError) {
-    SessionState.getConsole().getOutStream().println("afterExecution: " + ctx.getCommand());
-  }
-}
diff --git a/ql/src/test/queries/clientpositive/query_lifetime_hook.q b/ql/src/test/queries/clientpositive/query_lifetime_hook.q
index 8fda055..fed8031 100644
--- a/ql/src/test/queries/clientpositive/query_lifetime_hook.q
+++ b/ql/src/test/queries/clientpositive/query_lifetime_hook.q
@@ -1,3 +1,3 @@
-SET hive.query.lifetime.hooks=org.apache.hadoop.hive.ql.hooks.TestQueryLifeTimeHook;
+SET hive.query.lifetime.hooks=org.apache.hadoop.hive.ql.hooks.ConsoleQueryLifeTimeHook;
 
 SELECT * FROM src LIMIT 1;
-- 
1.7.9.5

